main.py

```diff
@@ -11,9 +11,15 @@
 from hashlib import sha1
 from pathlib import Path
 
-from pyftpdlib.authorizers import DummyAuthorizer
-from pyftpdlib.handlers import FTPHandler
-from pyftpdlib.servers import ThreadedFTPServer
+try:  
+    import pyftpdlib
+except ModuleNotFoundError:  
+        print ("Please pip install pyftpdlib==1.5.4" )
```

Comment:
Move entire try catch to __init__ of server

pyftpdlib is only required for server, not client